src/components/Explore/utils.ts

```diff
@@ -325,3 +336,55 @@ export function getAutocompleteLocations(locationFips: string) {
         .filter(isCounty)
         .filter(location => belongsToState(location, stateFips));
 }
+
+const SERIES_COLORS = [
+  '#1f77b4',
+  '#ff7f0e',
+  '#2ca02c',
+  '#d62728',
+  '#9467bd',
+  '#8c564b',
+  '#e377c2',
+  '#7f7f7f',
+  '#bcbd22',
+  '#17becf',
+];
+
+function getSmoothedSeries(
```

Comment:
nit: This should maybe be closer to getSeries() and maybe we should add comments to explain / differentiate the functions (and/or come up with clearer names)?

Feel free to punt for now.